lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
My thinking was that it may make the execution flow clearer since `report-template.html` is used for the report and viewer but this script is only for report, not viewer